app/containers/TrackDetailsContainer/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * TrackDetailsContainer
+ *
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { Card, Typography, Space, Avatar } from 'antd';
+import { musicContainerCreators } from '../MusicContainer/reducer';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    max-width: ${props => props.maxwidth}px;
+    width: 100%;
+    margin: 0 auto;
+    padding: ${props => props.padding}px;
+  }
+`;
+
+export function TrackDetailsContainer(props) {
+  const { track, dispatchGetTrack } = props;
+  const { trackId } = props.match.params;
+
+  useEffect(() => {
```

Comment:
we should first check if the track is already available in the store, if it isnt only then make an API call